drivers/volume/portworx/portworx.go

```diff
@@ -477,21 +462,11 @@ func (d *portworx) ValidateCreateVolume(name string, params map[string]string) e
 	var token string
 	token = d.getTokenForVolume(name, params)
 	t := func() (interface{}, bool, error) {
-		vols, err := d.getVolDriver(token).Inspect([]string{name})
+		volumeInspectResponse, err := d.getVolDriver().Inspect(d.getContext(token), &api.SdkVolumeInspectRequest{VolumeId: name})
```

Comment:
I think this might have something to do with PX security enabled torpedo runs. Can't create/inspect/delete/list volumes without passing the security token.

Reply:
the token is required when security is enabled, it is empty otherwise